cvarlist.md

```diff
@@ -1,12 +1,8 @@
-```
```

Comment:
Please add the back ticks back

Reply:
New to pull requests, starting small, they're confusing. Hope I got it right. 

Adjusted the .cfg I run, which should add the ``` automatically from now. 

```
clear
con_logfile conlog.log
echo ```
version
echo ```
echo 
cvarlist_all
con_logfile ""
```